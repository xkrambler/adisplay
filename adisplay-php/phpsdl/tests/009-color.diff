030+ NULL
031+ object(SDL_Color)#2 (4) {
032+   ["r"]=>
033+   int(0)
034+   ["g"]=>
035+   int(255)
036+   ["b"]=>
037+   int(0)
038+   ["a"]=>
039+   int(64)
030- int(16)
031- int(61440)
032- int(3840)
033- int(240)
034- int(15)
035- int(356651010)
036- string(24) "SDL_PIXELFORMAT_RGBA4444"
037- int(373694468)
038- string(24) "SDL_PIXELFORMAT_RGBA8888"
039- Done
040+ }
041+ NULL
042+ NULL
043+ NULL
044+ 
045+ Warning: SDL_MasksToPixelFormatEnum() expects parameter 2 to be integer, object given in /data/dev/accord_display8th/adisplay/phpsdl/tests/009-color.php on line 14
046+ bool(false)
047+ string(23) "SDL_PIXELFORMAT_UNKNOWN"
048+ int(373694468)
049+ string(24) "SDL_PIXELFORMAT_RGBA8888"
050+ Done