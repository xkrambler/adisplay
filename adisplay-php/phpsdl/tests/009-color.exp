Color
object(SDL_Color)#1 (4) {
  ["r"]=>
  int(1)
  ["g"]=>
  int(63)
  ["b"]=>
  int(127)
  ["a"]=>
  int(255)
}
object(SDL_Color)#2 (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(255)
  ["b"]=>
  int(0)
  ["a"]=>
  int(64)
}
Name
string(23) "SDL_PIXELFORMAT_UNKNOWN"
string(24) "SDL_PIXELFORMAT_RGBA4444"
string(20) "SDL_PIXELFORMAT_YVYU"
string(23) "SDL_PIXELFORMAT_UNKNOWN"
Mask
bool(true)
int(356651010)
int(16)
int(61440)
int(3840)
int(240)
int(15)
int(356651010)
string(24) "SDL_PIXELFORMAT_RGBA4444"
int(373694468)
string(24) "SDL_PIXELFORMAT_RGBA8888"
Done