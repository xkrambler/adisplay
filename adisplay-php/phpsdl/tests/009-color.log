
---- EXPECTED OUTPUT
Color
object(SDL_Color)#1 (4) {
  ["r"]=>
  int(1)
  ["g"]=>
  int(63)
  ["b"]=>
  int(127)
  ["a"]=>
  int(255)
}
object(SDL_Color)#2 (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(255)
  ["b"]=>
  int(0)
  ["a"]=>
  int(64)
}
Name
string(23) "SDL_PIXELFORMAT_UNKNOWN"
string(24) "SDL_PIXELFORMAT_RGBA4444"
string(20) "SDL_PIXELFORMAT_YVYU"
string(23) "SDL_PIXELFORMAT_UNKNOWN"
Mask
bool(true)
int(356651010)
int(16)
int(61440)
int(3840)
int(240)
int(15)
int(356651010)
string(24) "SDL_PIXELFORMAT_RGBA4444"
int(373694468)
string(24) "SDL_PIXELFORMAT_RGBA8888"
Done
---- ACTUAL OUTPUT
Color
object(SDL_Color)#1 (4) {
  ["r"]=>
  int(1)
  ["g"]=>
  int(63)
  ["b"]=>
  int(127)
  ["a"]=>
  int(255)
}
object(SDL_Color)#2 (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(255)
  ["b"]=>
  int(0)
  ["a"]=>
  int(64)
}
Name
string(23) "SDL_PIXELFORMAT_UNKNOWN"
string(24) "SDL_PIXELFORMAT_RGBA4444"
string(20) "SDL_PIXELFORMAT_YVYU"
string(23) "SDL_PIXELFORMAT_UNKNOWN"
Mask
bool(true)
int(356651010)
NULL
object(SDL_Color)#2 (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(255)
  ["b"]=>
  int(0)
  ["a"]=>
  int(64)
}
NULL
NULL
NULL

Warning: SDL_MasksToPixelFormatEnum() expects parameter 2 to be integer, object given in /data/dev/accord_display8th/adisplay/phpsdl/tests/009-color.php on line 14
bool(false)
string(23) "SDL_PIXELFORMAT_UNKNOWN"
int(373694468)
string(24) "SDL_PIXELFORMAT_RGBA8888"
Done
---- FAILED
