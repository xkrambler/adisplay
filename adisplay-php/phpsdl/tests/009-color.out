Color
object(SDL_Color)#1 (4) {
  ["r"]=>
  int(1)
  ["g"]=>
  int(63)
  ["b"]=>
  int(127)
  ["a"]=>
  int(255)
}
object(SDL_Color)#2 (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(255)
  ["b"]=>
  int(0)
  ["a"]=>
  int(64)
}
Name
string(23) "SDL_PIXELFORMAT_UNKNOWN"
string(24) "SDL_PIXELFORMAT_RGBA4444"
string(20) "SDL_PIXELFORMAT_YVYU"
string(23) "SDL_PIXELFORMAT_UNKNOWN"
Mask
bool(true)
int(356651010)
NULL
object(SDL_Color)#2 (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(255)
  ["b"]=>
  int(0)
  ["a"]=>
  int(64)
}
NULL
NULL
NULL

Warning: SDL_MasksToPixelFormatEnum() expects parameter 2 to be integer, object given in /data/dev/accord_display8th/adisplay/phpsdl/tests/009-color.php on line 14
bool(false)
string(23) "SDL_PIXELFORMAT_UNKNOWN"
int(373694468)
string(24) "SDL_PIXELFORMAT_RGBA8888"
Done